### DISCUSSION TITLE
What is the procedure for configuring "flight=PFMultiContainer" in promptflow?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/promptflow#3402] [Q&A] [answered]
What is the procedure for configuring "flight=PFMultiContainer" in promptflow?

**Comment by 0mza987** [comment]
I saw the same question in this issue:
https://github.com/microsoft/promptflow/issues/3344#issuecomment-2143260371

@Bowen-Guo Could you please help answer in the linked issue thread and @rvishwakarma001 can also check the answer?

**Comment by Bowen-Guo** [comment, marked as answer]
Sure, please check the response in this issue: https://github.com/microsoft/promptflow/issues/3344
